_component/responsive-navigation/index.md

```diff
@@ -18,14 +18,24 @@ iframe_height: large
 
 {% include partials/html-tabs.md %}
 
-<h3>SCSS <span class="link"><a href="scss/component.scss" target="_blank">Download SCSS</a></span></h3>
+<h3>PostCSS <span class="link"><a href="scss/component.scss" target="_blank">Download CSS</a></span></h3>
```

Comment:
Set this to "Download PostCSS" to match the others

Reply:
Good catch! Thanks